{
  "uid" : "352eb5d847d3459",
  "name" : "downcasts 0",
  "fullName" : "Contract: SafeCast toUint8.downcasts 0",
  "historyId" : "921a699588ab171676706b2f96c1e885",
  "time" : {
    "start" : 1635782665970,
    "stop" : 1635782666533,
    "duration" : 563
  },
  "description" : "",
  "descriptionHtml" : "",
  "status" : "broken",
  "statusMessage" : "Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0441fa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))",
  "statusTrace" : "Error: Returned error: HTTPConnectionPool(host='solana', port=8899): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f23a0441fa0>: Failed to establish a new connection: [Errno -3] Temporary failure in name resolution'))\n                    at Context.<anonymous> (test/utils/math/SafeCast.test.js:9:43)\n                    at processImmediate (node:internal/timers:464:21)",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "suite",
    "value" : "Contract: SafeCast toUint8"
  }, {
    "name" : "testClass",
    "value" : "Contract: SafeCast toUint8"
  }, {
    "name" : "testMethod",
    "value" : "downcasts 0"
  }, {
    "name" : "package",
    "value" : "Contract: SafeCast toUint8"
  }, {
    "name" : "resultFormat",
    "value" : "allure1"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "352eb5d847d3459.json",
  "parameterValues" : [ ]
}